docs/02-for-app-authors/11-external-data-checker.md

```diff
@@ -1,7 +1,41 @@
-# External data checker
+# External Data Checker
 
-The Flatpak External Data Checker is a tool that can be used to check for updates to external data sources. It is designed to be used in conjunction with the [Flathub](https://flathub.org) build system, but can be used with any Flatpak repository.
+The [Flatpak External Data Checker](https://github.com/flathub/flatpak-external-data-checker) is a tool that can be used to check for updates to external data sources. It is designed to be used in conjunction with the [Flathub](https://flathub.org) build system, but can be used with any Flatpak repository.
 
-It can be found in the [flatpak-external-data-checker](https://github.com/flathub/flatpak-external-data-checker) repository.
+Check the project's README for addition information not covered here.
 
-Please refer to the readme in the repo for further assistance.
+## Configuration
```

Comment:
The custom x-checker workflow thing should be mentioned. Right now flathub's default set up only allows checking the master branch of the repo.

If it has multiple branches like baseapps or extensions or even applications with a beta and master branch - the way to run x-checker is to add a custom worflow and disable flathub's one.

Reply:
The contents here were copied over from `02-for-app-authors/06-maintenance.md` largely as is (with just some capitalization and grammar tweaks). I'm fine to add more detail, but that could probably be a follow-up since this is just content that was moved here.

Reply:
ok sure